assets/_sass/components/blog/home-page/_home-page.scss

```diff
@@ -190,6 +190,10 @@ article {
   position: relative;
   display: flex;
   flex-direction: column;
+  background-color: $primary-fill-color;
+  padding:16px;
+  margin: 8px;
+  border-radius: 8px;
```

Comment:
I would recommend wrapping this in the .homepage class, else it will apply to all .article-card divs (see stand alone articles get affected)
```
.homepage {
  .article-card {
    background-color: $primary-fill-color;
    background-color: $primary-fill-color;
    padding: 16px;
    margin: 8px;
    border-radius: 8px;
  }
}
```